onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/label/provider/LabelProvider.java

```diff
@@ -240,4 +257,8 @@ public void clearAndSet(Map<String, String> defaultLabels) {
     this.previouslyUsedLabels.clear();
     this.previouslyUsedLabels = defaultLabels;
   }
+
+  void setOntologyManager(OntologyManager ontologyManager) {
+  this.ontology = ontologyManager;
```

Comment:
Add indentation.

Reply:
Also, it seems strange to assign `ontologyManager` to the field named `ontology`.